content/en/running-programs/_index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Running CHT implementations"
+linkTitle: "CHT Implementations"
+weight : 2
+---
+
+There are various ways of running Community Health Toolkit implementations. It is important to understand the different models for integrating CHT tools into a community health program as discussed below.
+
+**Accompaniment approach**
+
+In this approach, Medic accompanies partners; including ministries of health, through their digital journey that may include designing, building, deploying and supporting CHT based tools for their unique health system needs. 
+
+**Community led implementations**
+
+This model leverages the robust technical expertise of the CHT community of practice. A network of local and regional technical organizations that have been onboarded on CHT are able to provide design, development, deployment and support services for implementing partners - including ministries of health. This network also contributes their learnings to the community through documentation, reference applications, CHT forum and CHT core improvements. Community led implementations emphasize on the importance of collaboration, local ownership and sustainability.
+
+**Research initiatives, innovations and studies**
+
+Through human centered design, research institutions and academia design, test, refine and deploy open source digital health innovations which are based on the CHT or other mature open source technologies. These innovations ultimately contribute knowlegde through publications. These implementations incubate and test concepts or interventions before large scale implementation. They run for a limited time and often have pre-determined objectives. 
+
+## Considerations for digital health implementations
+To run a successful digital health implementation, the following considerations are made:
+- Scope - Clear definition of the challenge, duration of implementation, implementation deliverables, and the change expected as a result of implementation
+- Technical requirements - infrastructure (servers - physical/ cloud) and hardware (devices such as smartphones/tablets, laptops/computers) requirements
+- Context - general understanding of the geography (country/region) in the following domains; legal and policy landscape, infrastruture penetration e.g. mobile and electricity connectivity, community health setup and maturity e.g cadres of community health workers, coverage and functionality
+- Human resource requirements - skills to build, implement and sustain the implementation
```

Comment:
Capitalisation is inconsistent here. Also it's easier to scan if you bold the key point, eg:

```suggestion
To run a successful digital health implementation, the following considerations are made:
- **Scope**. Clear definition of the challenge, duration of implementation, implementation deliverables, and the change expected as a result of implementation.
- **Technical requirements**. Infrastructure (servers - physical/ cloud) and hardware (devices such as smartphones/tablets, laptops/computers) requirements.
- **Context**. General understanding of the geography (country/region) in the following domains; legal and policy landscape, infrastructure penetration e.g. mobile and electricity connectivity, community health setup and maturity e.g cadres of community health workers, coverage and functionality.
- **Human resource requirements**. Skills to build, implement and sustain the implementation.
```